packages/async-operations/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@cumulus/async-operations",
+  "version": "3.0.0",
+  "description": "Cumulus Core internal async operations module",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "engines": {
+    "node": ">=12.18.0"
+  },
+  "scripts": {
+    "clean": "rm -f ./dist/*.js ./dist/*.d.ts",
+    "test": "../../node_modules/.bin/ava",
+    "test:coverage": "../../node_modules/.bin/nyc npm test",
+    "tsc": "../../node_modules/.bin/tsc",
+    "tsc:listEmittedFiles": "../../node_modules/.bin/tsc --listEmittedFiles",
+    "tsc:watch": "../../node_modules/.bin/tsc -w",
+    "watch-test": "../../node_modules/.bin/tsc-watch --onsuccess 'npm test'"
+  },
+  "ava": {
+    "files": [
+      "tests/**/*.js"
+    ],
+    "fail-fast": true,
+    "verbose": true,
+    "timeout": "2m"
+  },
+  "author": "Cumulus Authors",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@cumulus/aws-client": "3.0.0",
+    "@cumulus/db": "3.0.0-alpha.1",
+    "@cumulus/logger": "3.0.0",
```

Comment:
I don't think the logger is used in here?

Reply:
4ef7beb36

Reply:
4ef7beb36